src/index.js

```diff
@@ -169,6 +176,7 @@ class Service {
   }
 
   find (params) {
+    debug('method find of service %s called', this.table);
```

Comment:
Should be removed for consistency (we might want to add more debug statements to all database adapters but that would be a different issue).